compiler/modules/print.llvm/llvm.bal

```diff
@@ -650,7 +671,11 @@ function sameIntType(Value v1, Value v2) returns IntType {
 function typeToString(RetType ty) returns string {
     string typeTag;
     if ty is PointerType {
-        typeTag = typeToString(ty.pointsTo) + "*";
+        if ty.addressSpace == 0 {
+            typeTag = typeToString(ty.pointsTo) + "*";
+        } else {
+            typeTag = createLine([typeToString(ty.pointsTo), "addrspace", "(" , ty.addressSpace.toString(), ")*"]);
```

Comment:
I think you need to split `")*"` into `")", "*"` so `omitSpaceBefore` can leave out the extra space (which @KavinduZoysa pointed out)

Reply:
Fixed